src/lib/modules/Literature/LiteratureCover.js

```diff
@@ -28,7 +29,10 @@ class LiteratureCover extends Component {
     } = this.props;
     const Cmp = asItem ? Item.Image : Image;
     const link = linkTo ? { as: Link, to: linkTo } : {};
-    return url ? (
+    const finalUrl = url
+      ? url
+      : invenioConfig.APP.INVENIO_UI_URL + '/api/static/images/placeholder.png';
```

Comment:
from what I remember it is done automatically by the backend, and the backend is responsible for adding the right placeholder. We shouldn't hardcode the placeholders, it is on the future developer to decide what to display